docs/source/releases/v2.1.rst

```diff
@@ -115,6 +115,9 @@ Minor changes
 - Fixed the logic of ``StockRequired.parent_availability_policy`` to use
   child products to determine availability of children, rather than the parent.
 
+- ``customer.forms.PasswordResetForm`` now uses the parent class' ``get_users()``
+  method to determine the list of users to send an email to.
```

Comment:
I think you should note that the behaviour changes here, i.e., unicode safe comparison and checking that the user has a usable password.